registration/src/main/java/com/kyriba/registration/api/TicketController.java

```diff
@@ -0,0 +1,49 @@
+package com.kyriba.registration.api;
+
+
+import com.kyriba.registration.api.dto.ChangeTicketOwnerRequest;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.Value;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+
+@Api(value = "Ticket management endpoint")
+@RestController
+@RequestMapping("/api/v1/tickets")
+public class TicketController
+{
+
+  /**
+   * UCS: Attendee: 4. change attendee
+   *
+   * @param exchange input API DTO for exchanging ticket
+   * @return ticket output API DTO
+   */
+  @ApiOperation(value = "Change ticket owner to an other attendee", response = TicketResponse.class)
```

Comment:
response is not needed because Swagger understands WebMVC annotations.

Reply:
removed